pages/login/index.js

```diff
@@ -1,33 +1,62 @@
-import axios from 'axios'
 import React, { useState, useEffect } from 'react'
 import Image from 'next/image'
 import { useDispatch, useSelector } from 'react-redux'
-import { useRouter } from 'next/router'
-import { changeLoginType, getUser, login } from 'redux/reducers/authSlice'
+import Router from 'next/router'
+import { getSession } from 'next-auth/react'
+import { loginUser } from 'redux/reducers/authSlice'
+import BannerModal from 'components/modals/BannerModal'
+import { isValidEmail } from 'utils/string'
 
-const Index = () => {
-  const router = useRouter()
+const Login = () => {
   const dispatch = useDispatch()
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
+  const [message, setMessage] = useState('')
+  const [error, setError] = useState(false)
+  const [showModal, setShowModal] = useState(false)
 
-  const { user, loginSuccess, loginType } = useSelector((state) => state.auth)
+  const { loading } = useSelector((state) => state.auth)
 
-  const signupInstead = () => {
-    router.push('/signup')
-  }
+  useEffect(() => {
+    if (showModal) {
+      const timer = setTimeout(() => {
+        setModal('', false, false)
```

Comment:
Why do you need to setTimeout here? @micohalvarez

Reply:
this is to autohide the modal 2 seconds after showing

Reply:
Why do we need to autohide? haha

Reply:
UX